### PR TITLE
fix issue-3773

### PR DESCRIPTION
**Pull request by Vips19**

### CODE REVIEW

**Comment by Vips19** [comment]
#3773

**Comment by Vips19** [comment]
@maxceem I have pushed the code. Could you please verify?

**Comment by Vips19** [comment]
@bug-bash-helper #3910 is ready for review

File: src/components/TopBar/NewProjectNavLink.jsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ const NewProjectNavLink = ({ compact=false, link }) => {
   } else {
     return (
       <div className="new-project-link">
-        <Link
+        <Link
```
Comment (maxceem) [inline src/components/TopBar/NewProjectNavLink.jsx:39]: @Vips19 can we revert unnecessary changes?

File: src/projects/create/components/ProjectSubmitted.jsx [hunk 2]
```diff
@@ -41,9 +41,9 @@ class ProjectSubmitted extends React.Component {
           <div className="project-link-container flex row middle center">
             { `${this.state.domain}${this.state.url}` }
           </div>
-          <div className="button-container flex row middle center">
+          <div className="button-container flex row middle">
             <a type="button" onClick={this.copyToClipboard} className="copy-link-btn tc-btn tc-btn-sm tc-btn-default flex middle center" disabled={false}>Copy link</a>
-            <a href={this.state.url} type="button" className="go-to-project-dashboard-btn tc-btn tc-btn-sm tc-btn-primary flex middle center" disabled={false}>Go to project dashboard</a>
+            <a className="go-to-project-dashboard-btn tc-btn tc-btn-sm tc-btn-primary flex middle center" href={this.state.url} type="button"  disabled={false}>Go to project dashboard</a>
```
Comment (maxceem) [inline src/projects/create/components/ProjectSubmitted.jsx:46]: @Vips19 can we revert unnecessary changes?

File: src/projects/create/components/ProjectSubmitted.jsx [hunk 3]
```diff
@@ -41,9 +41,9 @@ class ProjectSubmitted extends React.Component {
           <div className="project-link-container flex row middle center">
             { `${this.state.domain}${this.state.url}` }
           </div>
-          <div className="button-container flex row middle center">
+          <div className="button-container flex row middle">
```
Comment (maxceem) [inline src/projects/create/components/ProjectSubmitted.jsx:44]: @Vips19 is removing `center` class necessary for the fix?

File: src/projects/detail/containers/DashboardContainer.jsx [hunk 4]
```diff
@@ -274,7 +274,7 @@ class DashboardContainer extends React.Component {
             <ProjectPlanEmpty isManageUser={isManageUser} />
           )}
           {isProjectLive && hasPermission(PERMISSIONS.MANAGE_PROJECT_PLAN)  && !isLoadingPhases && (<div styleName="add-button-container">
-            <Link to={`/projects/${project.id}/add-phase`} className="tc-btn tc-btn-primary tc-btn-sm action-btn">Add New Phase</Link>
+            <Link to={`/projects/${project.id}/add-phase`} className="tc-btn tc-btn-primary tc-btn-sm action-btn" >Add New Phase</Link>
```
Comment (maxceem) [inline src/projects/detail/containers/DashboardContainer.jsx:277]: @Vips19 can we revert unnecessary changes?

File: src/projects/list/components/ProjectsTopBar/ProjectsTopBar.js [hunk 5]
```diff
@@ -6,7 +6,7 @@ const ProjectsTopBar = () => (
 
   <div className="topbar-container">
     <div className="topbar">
-      <Link to="/new-project"  className="logo tc-btn-sm tc-btn-primary">+ New Project</Link>
+      <Link className="logo tc-btn-sm tc-btn-primary" to="/new-project" >+ New Project</Link>
```
Comment (maxceem) [inline src/projects/list/components/ProjectsTopBar/ProjectsTopBar.js:9]: @Vips19 can we revert unnecessary changes?

File: src/styles/main.scss [hunk 6]
```diff
@@ -7,4 +7,16 @@ body {
...
+  
+  :global {
+    .tc-btn-primary:hover {
+      background: #3996FF;
+      background-image: -webkit-gradient(linear, left top, left bottom, from(#3996FF), to(#127BF3));
+      background-image: linear-gradient(#3996FF 0%, #127BF3 100%);
+    }
+    
+    .tc-btn-primary:hover,.tc-btn-primary:active,.tc-btn-primary:focus {
+      color: #ffffff;
```
Comment (maxceem) [inline src/styles/main.scss:19]: @Vips19 let's use a variable instead of color value as we do for button style https://github.com/appirio-tech/tc-ui/blob/feature/connectv2/src/styles/_buttons.scss#L69.

File: src/styles/main.scss [hunk 7]
```diff
@@ -7,4 +7,16 @@ body {
   box-sizing: border-box;
   // Prevenet jumping of elements, force scrollbar always-on
   overflow-y: scroll;
+  
+  :global {
+    .tc-btn-primary:hover {
+      background: #3996FF;
+      background-image: -webkit-gradient(linear, left top, left bottom, from(#3996FF), to(#127BF3));
+      background-image: linear-gradient(#3996FF 0%, #127BF3 100%);
```
Comment (maxceem) [inline src/styles/main.scss:15]: @Vips19 let's reuse the mixing the same way we do for button style https://github.com/appirio-tech/tc-ui/blob/feature/connectv2/src/styles/_buttons.scss#L73.

File: src/components/TopBar/NewProjectNavLink.jsx (hunk 1 above)
Comment (Vips19) [reply src/components/TopBar/NewProjectNavLink.jsx:39]: @maxceem Yes sure, sorry for the trouble

File: src/projects/create/components/ProjectSubmitted.jsx (hunk 2 above)
Comment (Vips19) [reply src/projects/create/components/ProjectSubmitted.jsx:46]: Yes sure

File: src/projects/create/components/ProjectSubmitted.jsx (hunk 3 above)
Comment (Vips19) [reply src/projects/create/components/ProjectSubmitted.jsx:44]: With my previous fix it was needed .. i will revert this change also

File: src/projects/detail/containers/DashboardContainer.jsx (hunk 4 above)
Comment (Vips19) [reply src/projects/detail/containers/DashboardContainer.jsx:277]: yes sure

File: src/projects/list/components/ProjectsTopBar/ProjectsTopBar.js (hunk 5 above)
Comment (Vips19) [reply src/projects/list/components/ProjectsTopBar/ProjectsTopBar.js:9]: @maxceem can i raise a new branch with the necessary changes only

Comment (maxceem) [reply src/projects/list/components/ProjectsTopBar/ProjectsTopBar.js:9]: @Vips19, yes, you can create a new pull request if you want.

File: src/styles/main.scss (hunk 7 above)
Comment (Vips19) [reply src/styles/main.scss:15]: Screenshot (136)

Are these changes fine? @maxceem I am not able to find the variable for #3996FF, #127BF3 as used in https://github.com/appirio-tech/tc-ui/blob/feature/connectv2/src/styles/_buttons.scss#L73. Can I leave the  @include background-gradient(#3996FF, #127BF3); part as it is or need to change with some variable??

Comment (maxceem) [reply src/styles/main.scss:15]: @Vips19 it's ok to keep color values for these, as we don't have variables for them.

Regarding the changes, as I can see for buttons we define color for 
```  
   &:hover,
    &:active,
    &:focus
```

and background gradient only for:
```
&:hover
```

While on your screenshot you make vise-verse.

Comment (Vips19) [reply src/styles/main.scss:15]: @maxceem , I have made changes according to the https://github.com/appirio-tech/tc-ui/blob/feature/connectv2/src/styles/_buttons.scss#L66. please let me know if its ok or I need to change it?

Comment (maxceem) [reply src/styles/main.scss:15]: @Vips19 sure, where can I check it?

Comment (Vips19) [reply src/styles/main.scss:15]: Line 66-72 in the _button.scss

Comment (maxceem) [reply src/styles/main.scss:15]: I'm not sure where you made changes. _button.scss is inside another repository

Comment (Vips19) [reply src/styles/main.scss:15]: @maxceem I have made changes according to _button.scss. Let me raise a PR then you can check it

Comment (Vips19) [reply src/styles/main.scss:15]: #3933

**Comment by maxceem** [comment]
Closing this PR as there is a new one https://github.com/appirio-tech/connect-app/pull/3933
